dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -58,13 +58,9 @@ if [ -z "$STARTED_BY_TSA_TSB_SERVICE" ]; then
 fi
 
 /usr/bin/TS TSA
-if [[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.type)" == *"SpineRouter"* ]] ; then
-  if [[ "$1" != "chassis" ]] ; then
-      echo "Please execute 'sudo config save' to preserve System mode in Maintenance after reboot or config reload"
-      if [[ $disaggregated_chassis -ne 1 ]]; then
-          echo -e "\nWARNING: Please execute 'TSA' on all other linecards of the chassis to fully isolate this device"
-      fi
-  fi
-else
-  echo "Please execute 'sudo config save' to preserve System mode in Maintenance after reboot or config reload"
-fi
```

Comment:
why remove this?

Reply:
line 62 was only applicable for rexec-based TSA where we were passing "TSA chassis" from sup. It is no longer needed.
Other changes are just making that warning message print on modular chassis alone by looking for upstream/downstreamLC

Reply:
TSA from sup can still be done by DRI on sup

Reply:
@rlhui - no functional change to TSA from sup in this PR. This is just code cleanup while printing warning message on linecard. Instead of using Spinerouter to identify linecard, we have switched to using subtype UpsteamLC/DownstreamLC